Lib/test/test_locale.py

```diff
@@ -390,6 +391,7 @@ def test_strcoll_with_diacritic(self):
         is_emscripten or is_wasi,
         "musl libc issue on Emscripten/WASI, bpo-46390"
     )
+    @unittest.skip("TODO: RUSTPYTHON")
```

Comment:
Does this work?
```suggestion
    # TODO: RUSTPYTHON
    @unittest.expectedFailure
```

Reply:
I fixed in https://github.com/RustPython/RustPython/pull/4558/commits/9e5be11c43806d2a4faa42a9348b47cbaa2d03c6